_targets/build.project.armv7m7-imxrt106x

```diff
@@ -38,49 +36,53 @@ export PORTS_CURL=y
 b_build_target() {
 	mkdir -p "$PREFIX_ROOTFS"/dev "$PREFIX_ROOTFS"/local "$PREFIX_ROOTFS"/data "$PREFIX_ROOTFS"/mnt "$PREFIX_ROOTFS"/var
```

Comment:
shorter
```
	mkdir -p "$PREFIX_ROOTFS/"{dev,local,data,mnt,var}
```

Reply:
Not a topic for this PR, but fixed anyway - IMHO creating empty dirs should be done as a part of `root-skel` (this is part of the skeleton, but keeping empty dirs in git repo is problematic) - so fixed globally in `phoenix-rtos-build/build.sh` as You've suggested

Fixed in: https://github.com/phoenix-rtos/phoenix-rtos-build/commit/a7185003e33388cd075a2a6e32c78ad2d6c7bc23